bin/pre_compile

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Syntax bin/compile <build-dir> <cache-dir>
+
+echo "-----> Writing out SOURCE_VERSION to static/hash ($SOURCE_VERSION)"
+echo $SOURCE_VERSION > $BUILD_DIR/staticfiles/hash
```

Comment:
Can you change this to something like `hash.html`? Right now it just downloads the thing which is less than ideal.